rbc/stdlib/array_attributes.py

```diff
@@ -0,0 +1,7 @@
+"""
+Array API specification for array attributes.
+
+https://data-apis.org/array-api/latest/API_specification/array_object.html#attributes
+"""
+
+__all__ = []
```

Comment:
Same. This is supposed to be `rbc/stdlib/data_type_functions.py` no?

Reply:
Same. https://data-apis.org/array-api/latest/API_specification/array_object.html#attributes